### PR TITLE
config: generic-th3d-ezboard-lite-v2.0.txt

### PR DESCRIPTION
**Pull request by Apulo**
module: GENERIC-TH3D-EZBOARD-LITE-V2.0 configuration

This file includes the build instructions and generic configuration for the TH3D EZBoard Lite v2.0 3d printer control board.

Signed-off-by: Anthony Dellett (anthony.dellett@gmail.com)

### CODE REVIEW

**Comment by Apulo** [comment]
> Thanks. In general, it looks fine to me, but I did notice a couple of items:
> 
> 1. This file should be added to the build test cases ( https://www.klipper3d.org/Example_Configs.html - point 4).
> 2. Using a `hold_current` is discouraged and I don't think any of the example configs should specify it (unless there are test results showing it is needed).  https://www.klipper3d.org/TMC_Drivers.html
> 3. The board_pins section is not deprecated.  It is a good idea to specify a board_pins section and it should not be commented out.
> 
> -Kevin

Thanks Kevin, I have made the suggested changes.

File: config/generic-th3d-ezboard-lite-v2.0.cfg [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+#[filament_switch_sensor my_sensor]
+#switch_pin: PC0
+
+########################################
+# EXP1 / EXP2 (display) pins
+########################################
+
+[board_pins]
+aliases:
+ EXP1 header
```
Comment (KevinOConnor) [inline config/generic-th3d-ezboard-lite-v2.0.cfg:125]: This line looks like it should have been a comment.

-Kevin

Comment (Apulo) [reply config/generic-th3d-ezboard-lite-v2.0.cfg]: You are correct. Sorry about that

**Comment by KevinOConnor** [comment]
Also, I don't see this PR updating the build test cases ( https://www.klipper3d.org/Example_Configs.html - point 4).

-Kevin

**Comment by Apulo** [comment]
> Also, I don't see this PR updating the build test cases ( https://www.klipper3d.org/Example_Configs.html - point 4).
> 
> -Kevin

My apologies. I edited the file in the master repo accidentally.

**Comment by KevinOConnor** [comment]
Thanks.

-Kevin

**Comment by Apulo** [comment]
Thanks Kevin

**Pull request merged**